packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -8,16 +8,29 @@ import User from '../../../database/types/User'
 import generateUID from '../../../generateUID'
 import insertUser from '../../../postgres/queries/insertUser'
 import IUser from '../../../postgres/types/IUser'
+import {analytics} from '../../../utils/analytics/analytics'
 import segmentIo from '../../../utils/segmentIo'
 import addSeedTasks from './addSeedTasks'
 import createNewOrg from './createNewOrg'
 import createTeamAndLeader from './createTeamAndLeader'
 import isPatientZero from './isPatientZero'
 
-// no waiting necessary, it's just analytics
-const handleSegment = async (user: User, isInvited: boolean) => {
-  const {id: userId, createdAt, email, featureFlags, tier, segmentId, preferredName} = user
+const bootstrapNewUser = async (newUser: User, isOrganic: boolean) => {
+  const {id: userId, createdAt, preferredName, email, featureFlags, tier, segmentId} = newUser
   const domain = email.split('@')[1]
+  const isPatient0 = await isPatientZero(userId, domain)
+  const r = await getRethink()
+  const joinEvent = new TimelineEventJoinedParabol({userId})
+
+  await Promise.all([
+    r({
+      user: r.table('User').insert(newUser),
+      event: r.table('TimelineEvent').insert(joinEvent)
+    }).run(),
+    insertUser(newUser)
+  ])
+
+  // Identify the user so user properties are set before any events are sent
```

Comment:
This is an attempt to fix the issue where user identification happens after the `Account Created` event. For example, [this user](https://analytics.amplitude.com/parabol/project/224689/search/amplitude_id%3D472337852118?collapsed=true&sessionHandle=YOxmKx2_YOxmLBi_G35h6bW_22x_c56a5850-6e89-41b6-8acf-d72bac53aff7R&eventId=795b708b-461b-11ed-ba9e-49b6f46c1139) 's `Account Created` event is not tagged with isPatient0 property.